src/bin/migrate.rs

```diff
@@ -619,6 +619,9 @@ fn migrations() -> Vec<Migration> {
             "));
             Ok(())
         }),
+        Migration::run(20160202210722,
+                       "UPDATE keywords SET keyword = LOWER(keyword)",
+                       "UPDATE keywords SET keyword = keyword"),
```

Comment:
Unfortunately this migration will trigger a violation when run over crates.io, for example the keyword `mouse` exists as well as `Mouse`, and this will cause the two to conflict.

I think the migration will have to basically handle each keyword in turn and custom merge it into another keyword if it exists.